auto-docs: Update @redpanda-data/docs-extensions-and-macros (#1684)
redpanda-data/documentation: docs/deploy/deployment-option/cloud/create-byoc-cluster-aws.mdx

8. Click **Next**.

    The **Networking** page opens.

> 9. Under **Choose Connection Type**, select either **Public** or **Private**. For BYOC clusters, **Private** is best-practice.
> 10. Your network name will be used in the Redpanda UI to identify this network.

Reviewer: ```suggestion
9. Under **Choose Connection Type**, select either **Public** or **Private**. For BYOC clusters, **Private** is best-practice. Your network name will be used in the Redpanda UI to identify this network. For a CIDR range, choose one that does not overlap with your existing GCP VPCs or your Redpanda network.
```

> 11. For a CIDR range, choose one that does not overlap with your existing GCP VPCs or your Redpanda network. 

Reviewer: ```suggestion

```

8. Click **Next**.

    The **Networking** page opens.

> 9. Under **Choose Connection Type**, select either **Public** or **Private**. For BYOC clusters, **Private** is best-practice. Your network name will be used in the Redpanda UI to identify this network. For a CIDR range, choose one that does not overlap with your existing GCP VPCs or your Redpanda network.

Reviewer 2: @micheleRP this is the GCP page. I assume you meant 'existing AWS VPCs'?

Reviewer 3: right, @ivpanda, I just put the comment here. On the [AWS page for Create a BYOC cluster](https://deploy-preview-1684--redpanda-documentation.netlify.app/docs/deploy/deployment-option/cloud/create-byoc-cluster-aws/#create-a-byoc-cluster), look at step 11: it says GCP.

    * `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY`
* Ensure that your role has the necessary permissions to run commands and create clusters.

> 

Reviewer 3: @ivpanda in the above section, instead of **+ Add Cluster**, the button now says **Create cluster**. And in the intro, would you please change "Bring your own cloud" to "Bring Your Own Cloud"? Same edits for the GPC page!

Reviewer 2: Change where, sorry? @micheleRP

Reviewer 3: it's up above, in Create a BYOC cluster section, step 1

> ## VPC Peering 

Reviewer 3: ```suggestion
## Next steps
```

> In order to start sending data to the Redpanda cluster, the networking process has to be completed. Your Redpanda VPC must be networked to your existing AWS VPC. To do that, continue to the next step of BYOC setup: 
> 
> * [VPC peering an AWS BYOC cluster](vpc-peering-aws.mdx)

Reviewer 3: ```suggestion
In order to start sending data to the Redpanda cluster, the networking process has to be completed. Your Redpanda VPC must be networked to your existing AWS VPC. To do that, continue to the next step of BYOC setup: 

* [Create a VPC Peering Connection to an AWS BYOC Cluster](vpc-peering-aws.mdx)
```
